### PR TITLE
package.json commit for npm-scriptpackage

### PR DESCRIPTION
**Pull request by DheepaKumar123**
Hi, please add this in cdnjs. I think it will be helpful for all.

Library name: npm-scriptpackage
 Git repository (required): https://github.com/DheepaKumar123/NpmScript.git
NPM package url (optional):https://www.npmjs.com/package/npm-scriptpackage

### CODE REVIEW

**Comment by extend1994** [comment]
Hi @DheepaKumar123 I'm afraid this lib doesn't meet out popularity requirement (200 stars/watchers on GitHub and 800 downloads/month on npm registry are good examples, see [issue rule](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-request-a-new-library) for more details). So I will close this PR if it's fine with you.

**Comment by DheepaKumar123** [comment]
Hi 

In peterbot test failed due to provided filename not exist in your repository. but, once you add my files library its will be available. then why its show like failed and its my first request for add new library.

Can you suggest solution for above issues.

Regards,
Dheepa Jhon

File: ajax/libs/npm-scriptpackage/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "npm-scriptpackage",
+  "filename": "blezzy.js",
+  "version": "14.2.29",
```
Comment (extend1994) [inline ajax/libs/npm-scriptpackage/package.json]: `version` should be removed.

File: ajax/libs/npm-scriptpackage/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "description": "Npm package of JavaScript.",
+  "keywords": [
+    "javascript"
+
```
Comment (extend1994) [inline ajax/libs/npm-scriptpackage/package.json]: L8, L9 should also be removed.

File: ajax/libs/npm-scriptpackage/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+   
+  ],
+  "author": "Dheepa Jhon",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:DheepaKumar123/NpmScript.git"
+  },
+  "license": "SEE LICENSE IN README.md",
```
Comment (extend1994) [inline ajax/libs/npm-scriptpackage/package.json]: Wrong format

File: ajax/libs/npm-scriptpackage/package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  "description": "Npm package of JavaScript.",
+  "keywords": [
+    "javascript"
+
+   
+  ],
+  "author": "Dheepa Jhon",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:DheepaKumar123/NpmScript.git"
```
Comment (extend1994) [inline ajax/libs/npm-scriptpackage/package.json]: https://github.com/DheepaKumar123/NpmScript is fine.

File: ajax/libs/npm-scriptpackage/package.json (hunk 3 above)
Comment (DheepaKumar123) [reply ajax/libs/npm-scriptpackage/package.json]: which one

File: ajax/libs/npm-scriptpackage/package.json (hunk 4 above)
Comment (DheepaKumar123) [reply ajax/libs/npm-scriptpackage/package.json]: But I faced issue while commit like that

**Comment by xtuc** [comment]
Please address the review

**Comment by MattIPv4** [comment]
Hi there!

This library doesn't have enough popularity (like more than 200 stars on GitHub or 800 downloads per month on npm). Please take a look at our [pull request template, essentials checklist](https://github.com/cdnjs/cdnjs/blob/master/PULL_REQUEST_TEMPLATE.md#essential-checklist) for more information regarding this. 📖

Please open an issue again when the library has more popularity, so that we can process it properly and add the library to cdnjs! Thanks for your great work and contribution, sorry we couldn't add your request at this time. 😄

**Pull request closed without merging**